app/questionnaire/placeholder_transforms.py

```diff
@@ -132,6 +134,21 @@ def format_number(self, number: Union[int, Decimal, str]) -> str:
 
         return ""
 
+    @staticmethod
+    def format_percentage(value: Union[int, Decimal, str]) -> str:
+        return f"{value}%"
+
+    @staticmethod
+    def format_unit(
+        unit: str, value: Union[int, Decimal, str], length: str = "short"
+    ) -> Any:  # babel units module not typed hence no specific return value
+        return units.format_unit(
+            value=value,
+            measurement_unit=unit,
+            length=length,
+            locale=flask_babel.get_locale(),
```

Comment:
`PlaceholderTransforms` already has a `locale` attribute, so this call to `get_locale` should be avoided.

Reply:
Using `PlaceholderTransform`s own locale now, thanks!